mitosheet/mitosheet/step_performers/graph_steps/plotly_express_graphs.py

```diff
@@ -118,21 +128,76 @@ def get_graph_creation_param_dict(
     if graph_type not in GRAPHS_THAT_DONT_SUPPORT_COLOR and color_column_header is not None:
         all_params['color'] = color_column_header
 
+    if facet_col_column_header is not None:
+        all_params['facet_col'] = facet_col_column_header
```

Comment:
Is the reason that we need to not set them if they are None is because in the transpiled code, we get parameters that are effectivley just set to defaults?

If so, I somehow feel like _filtering_ out in the transpile step might be nice. But this also might suck, because what if we have parameters where None is in fact not the default?

Reply:
Yeah the two reasons for this are: 
1) We don't want to include them in the transpiled code
2) I'm trying to keep the transpiled code and the plotly code we execute exactly the same so if there are small intricacies that we don't understand, we don't end up generating a different graph.